### PR TITLE
Sudhanshu/cart

### PR DESCRIPTION
**Pull request by SudhanshuGulhane**

### CODE REVIEW

**Review by shivamag00 — APPROVED** [review]

File: src/views/Home.vue [hunk 1]
```diff
@@ -32,7 +32,17 @@ export default {
...
-  emits : ["fetchData", "refreshNav"]
+  emits : ["fetchData", "refreshNav"],
+  data(){
+    return{
+      len:0,
+      lenp:0
+    }
+  },
+  mounted(){
+    this.len = Object.keys(this.categories).length
```
Comment (nilmadhab) [inline src/views/Home.vue]: use better variable names, like category_size and product_size

File: src/views/Product/ShowDetails.vue [hunk 2]
```diff
@@ -45,14 +54,15 @@ export default {
...
+      quantity: 1,
     }
   },
-  props : ["products", "categories"],
+  props : ["baseURL","products", "categories"],
   methods:{
     addToWishList(productId){
             console.log(this.isAddedToWishlist);
-            axios.post("http://remotedevs.org:8080/api/wishlist/add?token="+this.token, {
+            axios.post(this.baseURL+"wishlist/add?token="+this.token, {
```
Comment (nilmadhab) [inline src/views/Product/ShowDetails.vue:64]: make it `${this.baseURL}wishlist/add?token=${this.token}` for better readability

File: src/views/Product/ShowDetails.vue [hunk 3]
```diff
@@ -64,7 +74,38 @@ export default {
             },(error) =>{
                 console.log(error)
             });
+        },
+    addToCart(productId){
+      axios.post(this.baseURL+"cart/add?token="+this.token,{
+          productId : productId,
+          quantity : this.quantity
+      }).then((response) => {
+        console.log(response)
```
Comment (nilmadhab) [inline src/views/Product/ShowDetails.vue]: remove console.log

File: src/views/Product/ShowDetails.vue [hunk 4]
```diff
@@ -64,7 +74,38 @@ export default {
...
+          });
+          console.log("prod added!!!");
         }
+      },(error) =>{
+        console.log(error)
+      });
+    },
+
+    listCartItems(){
+      axios.get(this.baseURL+"cart/?token="+this.token).then((response) => {
```
Comment (nilmadhab) [inline src/views/Product/ShowDetails.vue:93]: same as above

**Review by nilmadhab — CHANGES_REQUESTED** [review]

File: src/views/Product/ShowDetails.vue (hunk 2 above)
Comment (nilmadhab) [reply src/views/Product/ShowDetails.vue:65]: use '`'

**Pull request closed without merging**